docs/source/markdown/podman-create.1.md

```diff
@@ -821,13 +821,15 @@ The following examples are all valid:
 
 Without this argument the command will be run as root in the container.
 
+**--userns**=*auto*
 **--userns**=*host*
 **--userns**=*keep-id*
 **--userns**=container:container
 **--userns**=*ns:my_namespace*
 
 Set the user namespace mode for the container.  It defaults to the **PODMAN_USERNS** environment variable.  An empty value means user namespaces are disabled.
 
+- `auto`: automatically create a namespace.  It is possible to specify other options to `auto`.  The only option supported currently is `size`, e.g. `--userns=auto:size=8192`.  If `size` is not specified, `auto` will automatically create a namespace with 4096 IDs available.
```

Comment:
Should probably reference the storage.conf setting for the 4096 IDs.  Also need to eventually mention that this is the greater of this number versus the minimum number required by the image.

Reply:
I've changed this part as it is not correct anymore.  I've done more changes on the containers/storage side.